functional-test/src/test/groovy/org/rundeck/tests/functional/selenium/BasicLoginSpec.groovy

```diff
@@ -19,7 +19,6 @@ class BasicLoginSpec extends SeleniumBase {
 
         then:
             currentUrl.contains("/menu/home")
-            pageSource =~ ~/\d+ Projects?/
     }
```

Comment:
why was this removed?

Reply:
it's better to validate against elements since pageSource gets evaluated before all of the javascript/jquery/ajax, and it wasn't working

Reply:
ah, ok.  I was trying to replicate the original test here https://github.com/rundeck/rundeck/blob/main/test/selenium/src/__tests__/selenium/login.test.ts#L31 but I was actually not doing it correctly anyway.  I will update it